coda/coda_mdstore/tests/test_resourcesync.py

```diff
@@ -0,0 +1,167 @@
+import pytest
+
+from django.contrib.sitemaps import Sitemap
+
+from .. import factories
+from .. import resourcesync
+
+pytestmark = [
+    pytest.mark.urls('coda_mdstore.urls'),
+    pytest.mark.django_db()
+]
+
+
+class FakeSitemap(Sitemap):
+    """Fake Sitemap object similar to resourcesync.BaseSitemap.
+
+    This will build but not persist model instances for testing
+    purposes.
+    """
+    lastmod = None
+    protocol = 'http'
+
+    def items(self):
+        # A miniumum of 50001 Bag objects are required to get multiple
```

Comment:
I do particularly like comments like this where it indicates why you used `5001` more than comments like "Check that all the dictionaries in `urlset` have the following keys" which more just say exactly what the code says (and that you probably would have thought too redundant to add if I hadn't mentioned adding comments). :)